app/components/global/AppBar/ProfileMenu.js

```diff
@@ -144,7 +144,7 @@ class ProfileMenu extends React.Component {
             <PopOverPanel>
               <Menu>
                 <MenuItem>
-                  <MenuHeading data-hj-suppress="">
+                  <MenuHeading data-hj-suppress="" data-test-id="profile-name">
```

Comment:
May be useful to also include a unit test for this as well as the browser test? there's already a `ProfileMenu.test.js` file so it would just need to shallow render this and assert the attribute is in the dom tree.

Reply:
done